example/install.bash

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# This script is used to run the ansible-galaxy iridl installation.
+# It will check for the existence of the virtual environment: /opt/datalib_venv3.12
+#
+# It will check for any local git changes that might need to be pushed, and it will check for
+# any git changes that need to be pulled.
+##
+VENV=/opt/datalib_venv3.12
+BUILD=""
+CHECK="-diff"
```

Comment:
Two hyphens, not one.

But --diff really has nothing to do with --check, so I wouldn't put it in the CHECK variable. Just hard-code --diff in the command in line 80.